src/utils/createCleanEvaluation.js

```diff
@@ -61,7 +63,10 @@ export function createCleanEvaluation() {
       id: "createdWith",
       value: packageJson.version,
     },
+    principleApplicability: principleApplicability
```

Comment:
add the object that contains applicability info to the `meta` section of our evaluation

Reply:
This is excellent @hidde